public/js/action-list.js

```diff
@@ -30,6 +30,14 @@
             this.on('close-column', this.onColumnClose, this);
 
             this.on('rendered', '.container', this.onRendered, this);
+            this.on('keydown','#body', this.onKeyDown, this);
```

Comment:
It should be possible to avoid listening on body, should it? I mean, listening on `#col1, #col2, .action-list` should also suffice, right?